internal/probes/worker.go

```diff
@@ -26,10 +26,13 @@ const (
 )
 
 type ProbeResult struct {
+	// CheckedAt the current helath check time
 	CheckedAt metav1.Time
```

Comment:
Do we need this value any more? I thought that the only that matters is `LastCheckedAt` in the `localProbe` :thinking: